### PR TITLE
YONK-16: Added split module store for tests

### PR DESCRIPTION
**Pull request by aamishbaloch**

### CODE REVIEW

**Comment by aamishbaloch** [comment]
@ziafazal Can you review this?

File: gradebook/tests.py [hunk 1]
```diff
@@ -64,7 +65,10 @@ def _get_homework_summary(self, course, attempted=False):
         return {
             u'url_name': u'Sequence_2',
             u'display_name': u'Sequence 2',
-            u'location': 'i4x://{org}/{num}/sequential/Sequence_2'.format(org=course.org, num=course.number),
+            u'location': 'block-v1:{org}+{num}+GRAD1+type@sequential+block@Sequence_2'.format(
```
Comment (ziafazal) [inline gradebook/tests.py]: `GRAD1` is course run if you can replace that with `course.run` as you did for org and number that would make it more flexible.

File: gradebook/tests.py [hunk 2]
```diff
@@ -76,7 +80,10 @@ def _get_midterm_summary(self, course, attempted=False):
         return {
             u'url_name': u'Sequence_3',
             u'display_name': u'Sequence 3',
-            u'location': 'i4x://{org}/{num}/sequential/Sequence_3'.format(org=course.org, num=course.number),
+            u'location': 'block-v1:{org}+{num}+GRAD1+type@sequential+block@Sequence_3'.format(
```
Comment (ziafazal) [inline gradebook/tests.py]: same here

**Review by ziafazal — CHANGES_REQUESTED** [review]
@aamishbaloch could you please make changes as suggested in comments.

**Comment by aamishbaloch** [comment]
@ziafazal Done with the change. Can you review it?

**Review by ziafazal — APPROVED** [review]
LGTM

**Pull request merged**